politeiawww/dcc.go

```diff
@@ -72,9 +71,7 @@ func createSponsorStatementRegex() string {
 			buf.WriteString(`\` + supportedChar)
 		}
 	}
-	buf.WriteString("]{")
-	buf.WriteString(strconv.Itoa(cms.PolicyMinSponsorStatementLength) + ",")
-	buf.WriteString(strconv.Itoa(cms.PolicyMaxSponsorStatementLength) + "}$")
```

Comment:
Don't you still need this max length in the regex?

Reply:
Well, actually no.  This is for regexp repeat checking and go limits that to 1000 (otherwise * always for infinite).  And we already check for min/max length prior to this.   So it's probably worth removing the other places that this is done throughout www.